processing/thermal.py

```diff
@@ -60,18 +60,51 @@ def process(recording, conf):
             classify(conf, recording, api, s3, logger)
 
 
-def classify(conf, recording, api, s3, logger):
-    working_dir = recording["filename"].parent
-    command = conf.classify_cmd.format(
-        folder=str(working_dir), source=recording["filename"].name
-    )
+def classify_models(api, command, conf):
+    """ classifies all models described in the config """
 
-    logger.debug("processing %s", recording["filename"])
+    model_results = []
+    main_model = None
+
+    for model in conf.models:
+        model_result = classify_file(api, command, conf, model=model)
+        if model.live:
+            main_model = model_result
```

Comment:
What if they forget to mark one model as the main model?   Should we assume the first one is live?